src/coreclr/jit/lsra.cpp

```diff
@@ -4217,7 +4202,7 @@ void LinearScan::unassignPhysReg(RegRecord* regRec, RefPosition* spillRefPositio
         return;
     }
 
-    regNumber victimAssignedReg = assignedInterval->physReg;
+    // regNumber victimAssignedReg = assignedInterval->physReg;
```

Comment:
should it be deleted instead?

Reply:
It is referenced in a TODO `#if 0` block below.